packages/solana/lib/src/wallet.dart

```diff
@@ -40,7 +40,6 @@ class Wallet {
     final signature = await _rpcClient.signAndSendTransaction(
       message,
       [signer],
-      commitment: commitment,
     );
     await _rpcClient.waitForSignatureStatus(signature, TxStatus.finalized);
```

Comment:
We should rather use `commitment` parameter and fallback to `TxStatus.finalized` if only it's not set.